arviz/tests/test_plots_matplotlib.py

```diff
@@ -156,6 +156,21 @@ def test_plot_trace_max_subplots_warning(models):
     assert axes.shape
 
 
+@pytest.mark.parametrize("kwargs", [{"var_names": ["mu", "tau"], "lines": [("hey", {}, [1])]}])
+def test_plot_trace_invalid_varname_warning(models, kwargs):
+    with pytest.warns(UserWarning):
```

Comment:
can you add a `match` argument too? maybe `match="valid var.+should be provided"`